estela-web/src/components/ProxySettingsPage/index.tsx

```diff
@@ -0,0 +1,295 @@
+        apiService.apiProxyProviderList(request).then((response: any) => {
+            response.results.forEach((item: any) => {
+                newBMCProxies.push({
+                    name: item.name,
+                    description: item.description,
+                    id: item.proxyid,
+                });
+            });
+            setBMCProxies(newBMCProxies);
+        });
+    };
+
+    useEffect(() => {
+        if (bmcProxies.length > 0) {
+            setOpenBMCModal(true);
+        }
+    }, [bmcProxies]);
+
+    const useBMCProxy = (proxyId: number): void => {
+        const proxySelected = bmcProxies.find((item) => item.id === proxyId);
+        setEnvVars([...envVars, {
+            name: "ESTELA_PROXY_NAME",
+            value: proxySelected ? proxySelected.name : "",
+            masked: false,
+        }])
+        setOpenBMCModal(false);
+    };
+
+    return (
+        <div>
+            {checkIfProxyExist() ? (
+                <Row className="bg-white rounded-lg my-4">
+                    <Space direction="vertical" className="lg:m-8 md:mx-6 m-4 w-full">
+                        <div className="flex flex-col">
+                            <p className="text-2xl text-black">Proxy Settings</p>
+                            <p className="text-estela-black-medium text-sm">
+                                Control and configure your proxies effortlessly.
+                            </p>
+                            <ProxyTag key="123" id={123}>
+                                {getProxyValue("ESTELA_PROXY_NAME")}
+                            </ProxyTag>
+                        </div>
+                    </Space>
+                </Row>
+            ) : (
+                <Row className="bg-white rounded-lg my-4">
+                    <Space direction="vertical" className="lg:m-8 md:mx-6 m-4 w-full">
+                        <div className="flex flex-col">
+                            <p className="text-2xl text-black">Proxy Settings</p>
+                            <p className="text-estela-black-medium text-sm">
+                                Control and configure your proxies effortlessly.
+                            </p>
+                            <div className="mt-4">
+                                <div className="">
+                                    <Row className="flex flex-col space-y-4">
+                                        <Button
+                                            onClick={() => {
+                                                setOpenProxyUserModal(true);
+                                            }}
+                                            size="large"
+                                            className="text-estela-blue-full w-96 h-24 border-0 bg-estela-blue-low text-base rounded estela-border-proxy"
+                                        >
+                                            <span className="text-center font-semibold">Manual configuration</span>
```

Comment:
Currently, the "Manual configuration" and other buttons are stacked on top of each other. I think the buttons should be next to each other in the same row to make use of the horizontal space. That is, instead of:
```
|Manual configuration|
|Other proxy|
```

It would be better to have:

```|Manual configuration| |Other proxy|```

Reply:
In the mockups developed for this feature, we use vertical alignment: https://www.figma.com/file/Z7B9w9UuqvO6hp0plhyCid/Bitmaker----ESTELA?node-id=21-204